config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
It always gets modified after running the controller. Is it safe to add this to `.gitignore`?

Reply:
oh... I see... the Makefile's `kustomize edit` is actually making this change.
I've been just reverting it manually before commits (w/o actually seeing what it does). You can leave it in.
We can't put it in gitignore though since we do need this file to be tracked.

Once it gets committed w/ the results of the edit, it should be fairly stable and not show up in PRs.